src/system/Form/Input.stories.jsx

```diff
@@ -36,3 +38,23 @@ export const Default = () => (
 		<Form.Input forLabel="input-with-custom-label" required />
 	</Form.Root>
 );
+
+export const WithFlexComponent = () => (
+	<Form.Root>
+		<Form.Input
+			placeholder="Your input here..."
+			label="Always add a label to inputs"
+			forLabel="input-simple"
+			flexComponent={ {
+				display: 'flex',
+				component: (
+					<Button sx={ { height: '40px' } } aria-label="copy">
```

Comment:
@kat3samsin not a blocker: could we add something to catch the button event, otherwise if we click it we are presented with an error page.

Reply:
Added copy button handler in https://github.com/Automattic/vip-design-system/pull/206/commits/95f7f92ceb118eca856fa29845daa60d5ec8bbd7